governance/polls/DAO Masons Poll Exercise 2 - README.md

```diff
@@ -0,0 +1,115 @@
+
+3. I would make the following forum response to the original thread:
+
+> [Governance Scope 2.6.6](https://mips.makerdao.com/mips/details/MIP101#2-6-6-aligned-delegate-privacy) defines a policy that requires me in my role as Governance Facilitator to take action when there is 'significant suspicion' that an AD's identity has been revealed. I believe what Diane has presented represents a 'significant suspicion' and so as a Facilitator I am bound to act.
+
+> According to [Governance Scope 2.6.5.1](https://mips.makerdao.com/mips/details/MIP101#2-6-5-1), FaciliatorDAO could immediately derecognize Clive as an AD. Due to the current lack of official FacilitatorDAOs described in governance to provide checks and balances, I've deemed it is more appropriate for Maker holders to make the determination in this case. In the future this decision may be made by FacilitatorDAOs unilaterally.
+
+> I've reached out to Clive to give him a chance to make a rebuttal, and will include links to his comments in the upcoming Poll. The poll is scheduled to begin on Monday July 31st.
+
+4. I would simultaneously reach out to Clive and let him know why I started a governance poll and ask him if he'd like to make a statement to be included in the poll.
+
+> Hello Clive,
+
+> I am reaching out regarding the recent allegations that your identity as an AD has been compromised. To be clear, I am following rules defined in [Governance Scope 2.6.6](https://mips.makerdao.com/mips/details/MIP101#2-6-6-aligned-delegate-privacy) that require me to take action when there is 'significant suspicion' that an AD's identity has been revealed. I believe what Diane has presented represents a 'significant suspicion' and so as a Facilitator I am bound to act.
+
+> While FaciliatorDAO could immediately derecognize you as an AD, as authorized by [Governance Scope 2.6.5.1](https://mips.makerdao.com/mips/details/MIP101#2-6-5-1), it seems more appropriate for Maker holders to make the determination in this case. Please see my latest reply in the forum thread for more detail.
+
+> As you are contesting the claims made against you, I wanted to be sure that you've had a chance to make your case and that your side of the story is heard.
+
+> In the governance poll, I'll be linking your reply to Diane's thread. If there is anything else you would like included I recommend you reply to that thread or create your own forum post and send me the link. I intend to put the poll up in 48 hours, so please provide anything you would like included before then.
+
+> Best,
+> DAO Masons
+> Interim Facilitator
+
+4. On Monday July 31 I would generate the governance poll (see other included file)
+
+   I've chosen to use a 50% majority poll for this poll because 50% of Maker holders signaling derecognition seems like a good signal of 'significant suspicion' to me.
```

Comment:
This is equivalent to plurality / binary, no? There are only two options: Yes or No. Would go with the simpler construct given this is the case. (Abstain votes don't count either for or against)